ferrocene/doc/qualification-plan/src/change-tracking.rst

```diff
@@ -26,15 +26,11 @@ to have parallel development on new targets or with specialized requirements.
 
 The Ferrocene toolchain maintains several artifacts that track the changes
 between **stable releases**. New features that have been introduced in a stable
-release are documented in features files. New KPs that have been identified and
-existing KPs that have been fixed in stable releases are documented in KP
-files. Once a new stable release is produced, the Ferrocene team sends a mass
-email to existing customers, notifying them of the release, its new features
-and KPs. In addition, customers and interested parties may view these artifacts
-online on the Ferrocene landing page.
+release are documented in the :doc:`release-notes:index`. New KPs that have
```

Comment:
feel free to ignore... just nice to introduce what KP stands for, since it's not a common abbreviation
```suggestion
release are documented in the :doc:`release-notes:index`. New Known Problems (KPs) that have
```

Reply:
`KP` will generate a link to the glossary though:

![image](https://github.com/ferrocene/ferrocene/assets/2299951/55e29ce9-6192-4c9e-9752-8a7a2b14340c)

Do you think it's still needed to define KP here?

Reply:
yeah, saw that... was just thinking would be nice to show the full words when first introduced

Reply:
This is not the first place it shows up in the documents... let's keep just the link.